docs/how-to-guides/traefik-ingress-example.md

```diff
@@ -0,0 +1,219 @@
+---
+title: Using the Traefik Ingress Controller
+---
+
+import TabsConstants from '@site/core/TabsConstants';
+
+<head>
+  <link rel="canonical" href="https://docs.rancherdesktop.io/how-to-guides/traefik-ingress-example"/>
+</head>
+
+Rancher Desktop uses `K3s` under the hood, which in turn uses [Traefik](https://doc.traefik.io/traefik/) as the default Ingress controller for your Kubernetes cluster. As an example, the below steps outline creating simple services that can be routed by the Ingress object.
+
+### Example Steps: Traefik Ingress Controller
+
+<Tabs groupId="os" defaultValue={TabsConstants.defaultOs}>
+<TabItem value="Linux">
+
+1. Set the node IP:
+
+  ```console
+  IP=$(kubectl get node/lima-rancher-desktop -o json | jq -r '.status.addresses[] | select(.type=="InternalIP").address')
```

Comment:
For Linux, as it uses Lima, the `IP` variable should be set to `127.0.0.1`.
This will allow to use the `host` address (i.e. whoami.$IP.nip.io) and be reachable from the host machine too.